fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -389,7 +389,16 @@ public Response createOrReplaceObjectRdf(
         }
 
         try {
-            session.save();
+            // prevent SN-Sibling nodes from creating.
+            synchronized (LOGGER) {
```

Comment:
@ajs6f Sorry maybe I need to explain that a user can use a program to submit the same ingest many times (see https://jira.duraspace.org/browse/FCREPO-2055), or click the HTML UI button many times to submit the same ingest with a malfunctioned browser?